Sources/NetworkProtection/Models/AnyIPAddress.swift

```diff
@@ -19,7 +19,7 @@
 import Foundation
 import Network
 
-public enum AnyIPAddress: IPAddress, Hashable, CustomDebugStringConvertible, @unchecked Sendable {
```

Comment:
This was causing issues because some IP addresses contained by this were converting to empty strings.  I'm removing the `IPAddress` protocol and consumers will instead need to call `ipAddress` on an instance of this class.